example-extension/composer.json

```diff
@@ -24,27 +24,29 @@
         "typo3/cms-core": "^11.5 || ^12.4"
     },
     "require-dev": {
-        "mediatis/typo3-coding-standards": "^1.0"
+        "mediatis/typo3-coding-standards": "^2.0",
+        "phpunit/phpunit": "^9.6.19",
+        "typo3/testing-framework": "^7.0.4"
```

Comment:
@mkarulin Is it necessary for the example package to have the requirements `phpunit/phpunit` and `typo3/testingframework`? The strategy so far was to add such dependencies implicitly via the package `mediatis/typo3-coding-standards`.